include/ddc/kernels/splines/bsplines_non_uniform.hpp

```diff
@@ -213,22 +213,22 @@ class NonUniformBSplines : detail::NonUniformBSplinesBase
         /** @brief Evaluates non-zero B-spline derivatives at a given coordinate
          *
          * The derivatives are computed for every B-spline with support at the given coordinate x. There are only (degree+1)
-         * B-splines which are non-zero at any given point. It is these B-splines which are derivated.
+         * B-splines which are non-zero at any given point. It is these B-splines which are derived.
```

Comment:
```suggestion
         * B-splines which are non-zero at any given point. It is these B-splines which are differentiated.
```
![image](https://github.com/user-attachments/assets/25dfb501-2bc0-43dd-844b-e196a4593395)

![image](https://github.com/user-attachments/assets/fd2d9322-14a1-4b49-be8d-00306bfdd6e0)

Reply:
Thanks!

Reply:
That said, the function documented is called `eval_deriv` :D

Reply:
Ah we talk about derivative the result of differentiating ?

Reply:
Yes but `deriv` is short for `derivative` not `derive`

Reply:
"evaluate the derivative"

Reply:
I understand

Reply:
Fixed in bf00e0181bc85431f65ecf9db269fde7a2ced09a